### PR TITLE
Support page wording

### PR DESCRIPTION
**Pull request by odscrachel**
Updating support page to include email addresses for OCP and CoST

### CODE REVIEW

File: docs/support.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ We can:
 * Provide guidance on mapping your data structures to the standard;
 * Give you feedback on draft data files, and support with validation of your data;
 
-Through funding from the Open Contracting Partnership and CoST - the Infrastructure Transparency Initiative, this support is provided free of charge.
+Use the following email addresses to request support:
 
-E-mail [data@open-contracting.org](mailto:data@open-contracting.org) with your questions, and a member of the OC4IDS Helpdesk will be in touch with you soon.
+* [opencode@infrastructuretransparency.org](mailto:opencode@infrastructuretransparency.org) for support from CoST - the Infrastructur Transparency Initiative
```
Comment (duncandewhurst) [inline docs/support.md]:
```suggestion
* [opencode@infrastructuretransparency.org](mailto:opencode@infrastructuretransparency.org) for support from CoST - the Infrastructure Transparency Initiative
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
After committing my suggestion, please request a review from @jpmckinney before merging. Thanks!

**Comment by duncandewhurst** [comment]
@odscrachel please could you redo this PR branching from `0.9` rather than `0.9-dev` so that we can release it without also releasing the changes on `0.9-dev`? Thanks!

**Comment by jpmckinney** [comment]
Closing in favor of #382. I can then merge 0.9 back into 0.9-dev.

**Pull request closed without merging**